kitsune/messages/models.py

```diff
@@ -5,7 +5,6 @@
 
 from kitsune.sumo.models import ModelBase
 
-
```

Comment:
Super nit. I think we do want this to be two empty lines after the imports?

Reply:
This is done automatically by `black`. It's going to introduce two lines after the imports and before the class definition but it won't change it for variable definitions.